torch/csrc/distributed/c10d/SymmetricMemory.hpp

```diff
@@ -71,6 +71,14 @@ class TORCH_API SymmetricMemory : public c10::intrusive_ptr_target {
 
   virtual int get_rank() = 0;
   virtual int get_world_size() = 0;
+
+  virtual std::vector<int> get_rank_to_global_rank() {
```

Comment:
is this NYI or NotImplemented?  (Will we implement it for Cuda backend?  should this be documented?  (why is this different than c10d existing rank mapping helpers)

Reply:
Yes, it stands for unimplemented. 
Good question whether other backends need it. Perhaps I should move them into `NVSHMEMSymmetricMemory` for now.